src/index.test.js

```diff
@@ -59,35 +46,37 @@ const differencify = new Differencify(globalConfig);
 
 describe('Differencify', () => {
   afterEach(() => {
-    loggerCalls = [];
-    writeFileSyncCalls = [];
-    chromyCloseCallsCounter = 0;
+    mockLog.mockClear();
+    logger.log.mockClear();
+  });
+  it('constructor fn', async () => {
+    expect(fs.mkdirSync).toHaveBeenCalledWith('screenshots');
+    expect(fs.mkdirSync).toHaveBeenCalledWith('./differencify_report');
   });
   it('update fn', async () => {
     const result = await differencify.update(testConfig);
     expect(result).toEqual(true);
-    expect(differencify.chromeInstancesId).toEqual(9223);
-    expect(loggerCalls[0]).toEqual('goto -> www.example.com');
-    expect(loggerCalls[1]).toEqual('capturing screenshot of whole DOM');
-    expect(loggerCalls[2]).toEqual('screenshot saved in -> screenshots/default.png');
-    expect(writeFileSyncCalls).toEqual(['screenshots', './differencify_report', 'screenshots/default.png', 'png file']);
+    expect(differencify.chromeInstances[3000]).toEqual(undefined);
```

Comment:
Also, it'd be better to check if this has been cleaned of all the instances rather than relying on the mock returning 3000. `expect(differencify.chromeInstances).toEqual({});`